### DISCUSSION TITLE
Crash to desktop on entering beatmap after editing skin

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#22322] [Q&A] [answered]
I edited the skin with the new elements, and when i played a beatmap (after already playing a couple), the game suddenly crashed.
I have since restarted the game, went straight to the beatmap and it played correctly, not sure what happened.

[database.log](https://github.com/ppy/osu/files/10473052/database.log)
[network.log](https://github.com/ppy/osu/files/10473053/network.log)
[performance.log](https://github.com/ppy/osu/files/10473050/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10473051/runtime.log)

**Comment by Loreos7** [comment]
Can reproduce. 
For me this only happens when you're adding `JudgementCounterDisplay`, saving the skin, retrying a beatmap, then deleting this new component.
[network.log](https://github.com/ppy/osu/files/10473140/network.log)
[performance.log](https://github.com/ppy/osu/files/10473141/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10473142/runtime.log)

**Reply by Theighlin** [reply]
I couldn't reproduce reliably so i didn't post any steps, could you share what you did so we can verify it's the same thing? 
I just added the new progress bar and hit results counter onto a legacy skin.
Also, the game is throwing some unhandled exceptions here and there, that i have noticed being the same error of the crash.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/22326
